Cargo.toml

```diff
@@ -38,8 +38,6 @@ rand = "0.8.4"
 term = "1.0.1"
 
 [features]
-# Use nightly-only features for a minor performance boost in PNG decoding
-unstable = ["crc32fast/nightly"]
```

Comment:
I think it might be worth keeping this feature around to enable the nightly feature on crc32fast (which enables an optimized implementation for aarch64)

Reply:
Good idea.  Done.

(I also retained this crate feature in `README.md` and `.github/workflows/rust.yml`;  OTOH, I felt that it's still okay to stop referring to this feature in `benches/unfilter.rs`.)